src/inputs.ts

```diff
@@ -1,34 +1,39 @@
-import { getInput, notice } from '@actions/core'
+import { getInput, notice, warning } from '@actions/core'
 import semver from 'semver'
+
 export interface Inputs {
   version: string | null
   venv: string | null
+  cache: boolean
 }
 
 export function getInputs(): Inputs {
+  const version = getVersionInput('uv-version')
   return {
-    version: getVersionInput('uv-version'),
-    venv: getVenvInput('uv-venv')
+    version,
+    venv: getVenvInput('uv-venv'),
+    cache: getCacheInput('uv-cache', version)
   }
 }
 
 export function getVersionInput(name: string): string | null {
   const version = getInput(name)
-  if (!version) {
-    notice('No uv version specified. Using latest version')
+  if (!version || version.toLowerCase() === 'latest') {
+    notice('Using latest uv version')
```

Comment:
Maybe I miss something, but if the previous print was already `notice` and it showed as warning (which is weird by itself), how does this solve the issue?
Thanks for addressing the issue